skel/README.md

```diff
@@ -1,3 +1,22 @@
 # Go-Gin web application
 
-This application was generated with [generator-gin](https://github.com/openware/generator-gin)
\ No newline at end of file
+This application was generated with [generator-gin](https://github.com/openware/generator-gin)
+
+# How to run sonic with client
+
+1. Put your frontend to client
+
+2. Run makefile
+```
+make asset
+```
+It will run build in client and after that all build files will moved to public/assets.
```

Comment:
It will run build in the `client` folder and then all build files will be moved to public/assets.